packages/backend/src/util/handlers/createExpressSynthqlHandler.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, test } from 'vitest';
+import { query } from '@synthql/queries';
+
+import { DB } from '../../tests/db';
+const from = query<DB>().from;
+
+describe('createExpressSynthqlHandler', () => {
+    test(`Query execution is successful`, async () => {
+        const q = from('actor')
```

Comment:
this isn't testing anything @jimezesinachi

Reply:
Correct! These are just the placeholder logic for whatever tests we decide to add for the handler function.